packages/dashboard/src/_stories/Dashboard.stories.tsx

```diff
@@ -36,6 +39,27 @@ const meta: Meta<typeof Dashboard> = {
 
 type Story = StoryObj<typeof Dashboard>
 
+export const Dashboard_Filter_Asc: Story = {

```

Comment:
seems like the only thing that is changing between these three configs is the order property. Can we just reuse the same config for all three of these and update that one property before it's passed in here?